distribution/docs/src/main/resources/_contents/_developing/developing-intro-contents.adoc

```diff
@@ -0,0 +1,816 @@
+
+== ${branding} Development Guidelines
+
+[NOTE]
+====
+Development requires full knowledge of the ${branding} Catalog.
+====
+
+${branding} is written in Java and requires a moderate amount of experience with the Java programming language, along with Java terminology, such as packages, methods, classes, and interfaces.
+${branding} uses a small OSGi runtime to deploy components and applications.
+Before developing for ${branding}, it is necessary that developers have general knowledge on OSGi and the concepts used within.
+This includes, but is not limited to, Catalog Commands and the following topics:
+
+* The Service Registry
+** How services are registered
+** How to retrieve service references
+* Bundles
+** Their role in OSGi
+** How they are developed
+
+Documentation on OSGi can be viewed at the http://www.osgi.org[OSGi Alliance website].
+Helpful literature for beginners includes _OSGi and Apache Felix 3.0 Beginner's Guide_ by Walid Joseph Gédéon and _OSGi in Action: Creating Modular Applications in Java_ by Richard Hall, Karl Pauls, Stuart McCulloch, and David Savage.
+
+=== Recommended Hardware
+
+Because of its modular nature, ${branding} may require a few or many system resources, depending on which bundles and features are deployed.
+In general, ${branding} will take advantage of available memory and processors.
+A 64-bit JVM is required, and a typical installation is performed on a single machine with 16GB of memory and eight processor cores.
```

Comment:
We probably need to be more specific here and say that we need the Oracle implementation or if using OpenJDK (or other) that extra security libraries need to be installed. @stustison?